src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/MotionDetailsPageEvent/MotionEventData.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { ColonyAndExtensionsEvents } from '~types';
+import {
+  getMotionEventTitleValues,
+  TransactionMeta,
+} from '~common/ColonyActions';
+import { useUserByNameOrAddress } from '~hooks';
+import EventData from '../EventData';
+
+import { MotionDetailsPageEventProps } from './MotionDetailsPageEvent';
+
+const displayName = 'common.ColonyActions.ActionDetailsPage.ActionEventData';
+
+type MotionEventDataProps = Pick<
+  MotionDetailsPageEventProps,
+  'actionData' | 'eventName' | 'motionMessageData'
+>;
+
+const MotionEventData = ({
+  actionData: { createdAt, transactionHash },
+  actionData,
+  motionMessageData,
+  eventName,
+}: MotionEventDataProps) => {
+  const { user: motionMessageInitiatorUser } = useUserByNameOrAddress(
+    motionMessageData.initiatorAddress || '',
+  );
```

Comment:
If you turn `getMotionEventTitleValues` into a hook, you can move this inside it and avoid passing an extra arg

Reply:
I do think it makes sense to use the `getUserByAddress` query here as well, as Jakub mentioned, because we know we're getting an address.